reference/import_export/product-export.rst

```diff
@@ -109,49 +82,63 @@ Product Processor
 
 This element receives product objects one by one, transforms each product object into an array and returns the array.
 
-The service is defined in ``src\Pim\Bundle\BaseConnectorBundle\Resources\config\processors.yml``.
+The service is defined in ``src/Pim/Bundle/ConnectorBundle/Resources/config/processors.yml``.
 
 .. code-block:: yaml
 
     parameters:
         pim_base_connector.processor.product_to_flat_array.class: Pim\Bundle\BaseConnectorBundle\Processor\ProductToFlatArrayProcessor
 
     services:
-        pim_base_connector.processor.product_to_flat_array:
-            class: %pim_base_connector.processor.product_to_flat_array.class%
+        pim_connector.processor.normalization.product:
+            class: '%pim_connector.processor.normalization.product.class%'
             arguments:
-                - '@pim_serializer'
-                - '@pim_catalog.manager.channel'
-                - ['pim_catalog_file', 'pim_catalog_image']
-                - %pim_catalog.localization.decimal_separators%
+                - '@pim_serializer.normalizer.product'
+                - '@pim_catalog.repository.channel'
+                - '@pim_catalog.repository.attribute'
+                - '@pim_catalog.builder.product'
+                - '@akeneo_storage_utils.doctrine.object_detacher'
+                - '@pim_connector.processor.bulk_media_fetcher'
 
-The class ``Pim\Bundle\BaseConnectorBundle\Processor\ProductToFlatArrayProcessor`` mainly delegates the transformation to the service ``pim_serializer``.
+The class ``Pim\Component\Connector\Processor\Normalization\ProductProcessor`` mainly delegates the transformation to the service ``pim_serializer.normalizer.product``.
 
-We can see here that we normalize each product into the ``flat`` format (= csv format).
+We can see here that we normalize each product into the ``standard`` format. It's writer's job to convert the standard format to the flat format.
```

Comment:
Don't hesitate to write sub task / post it on all missing parts you're seeing, it will be easier to split the work and get help :) I do agree that we can merge small PRs to update the doc and add missing parts before to do a global review (to ensure style and coherence). Chipmunks squad will be able to help to identify missing parts :)